disambiguate version and versions (#3825)
fishtown-analytics/docs.getdbt.com: website/snippets/_version-callout.md

:::important

> Take note that a [version tag](/reference/project-configs/version) (or `version`) is different to [`versions`](/reference/resource-properties/versions_) or [Model versions](/docs/collaborate/govern/model-versions). A version tag refers to the compatibility of the dbt project with a specific version of dbt. A version tag isn't required for dbt v1.5 and higher. 

Author: ```suggestion
Take note that a [version tag](/reference/project-configs/version) (or `version`) is different to [`versions`](/reference/resource-properties/versions) or [Model versions](/docs/collaborate/govern/model-versions). A version tag refers to the compatibility of the dbt project with a specific version of dbt. A version tag isn't required for dbt v1.5 and higher. 
```

> Take note that a [version tag](/reference/project-configs/version) (or `version`) is different to [`versions`](/reference/resource-properties/versions) or [Model versions](/docs/collaborate/govern/model-versions). A version tag refers to the compatibility of the dbt project with a specific version of dbt. A version tag isn't required for dbt v1.5 and higher. 

Author: ```suggestion
Take note that [model versions](/docs/collaborate/govern/model-versions) are different from [dbt_project.yml versions](/reference/project-configs/version#dbt_projectyml-versions) and [.yml property file versions](/reference/project-configs/version#yml-property-file-versions).

Model versions is a _feature_ that enables better governance and data model management by allowing you to track changes and updates to models over time. dbt_project.yml versions refer to the compatibility of the dbt project with a specific version of dbt. Version numbers within .yml property files inform how dbt parses those YAML files. The latter two are completely optional starting from version 1.5.
```

> Model versions and `versions` are _features_ that enable better governance and data model management by allowing you to track changes and updates to models over time.  Model versions and `versions` are available in dbt v1.5 or higher.

Reviewer: At a high-level, this does exactly what we hope by disambiguating between [model versions](https://deploy-preview-3825--docs-getdbt-com.netlify.app/docs/collaborate/govern/model-versions) and the [two other types of version](https://deploy-preview-3825--docs-getdbt-com.netlify.app/reference/project-configs/version)!

At a detailed level, I typically like `code` highlighting, but for some reason it is often hard for me to read within our docs site.

So this shorter alternative is more readable to me:
```suggestion
Take note that [model versions](/reference/resource-properties/versions) are different from [dbt_project.yml versions](/reference/project-configs/version#dbt_projectyml-versions) and [.yml property file versions](/reference/project-configs/version#yml-property-file-versions).
```

The suggestion above:
1. inverts the ordering of the different types
1. shortens the note by merely linking out to other docs (but not explaining further)
1. enumerates the three different types of `version`:
    1. model versions
    2. [dbt_project.yml versions](https://deploy-preview-3825--docs-getdbt-com.netlify.app/reference/project-configs/version#dbt_projectyml-versions)
    3. [.yml property file versions](https://deploy-preview-3825--docs-getdbt-com.netlify.app/reference/project-configs/version#yml-property-file-versions)

Reviewer: What do you think about the suggestion above @mirnawong1 ?

Author: hey @dbeatty10 ! have some thoughts on this and will reply after this meeting 🙏 was just catching up this week and updated the branch to remind me to address this pr today. thank you as always for being a 🪨⭐️

Author: hey @dbeatty10, i love your suggestion! i generally think it's useful to provide users with some context when explaining the difference. this is because i think it saves them reading and also may spark curiosity or interest. 

what do you think of 'meshing' the two so it reads (notice i replaced the 'model versions' link to the govern/model-versions one, is that right?):

> Take note that [model versions](/docs/collaborate/govern/model-versions) are different from [dbt_project.yml versions](/reference/project-configs/version#dbt_projectyml-versions) and [.yml property file versions](/reference/project-configs/version#yml-property-file-versions).
> 
> dbt_project.yml versions refer to the compatibility of the dbt project with a specific version of dbt. Model versions and versions are _features_ that enable better governance and data model management by allowing you to track changes and updates to models over time.

Reviewer: @mirnawong1 Makes sense that context is helpful 🧠 

Your proposal looks good, especially since the first sentence is a nice TLDR and the second paragraph is separated and gives more context.

The second paragraph is still a bit confusing to me. What do you think about the following?

> Take note that [model versions](/docs/collaborate/govern/model-versions) are different from [dbt_project.yml versions](/reference/project-configs/version#dbt_projectyml-versions) and [.yml property file versions](/reference/project-configs/version#yml-property-file-versions).
> 
> Model versions is a _feature_ that enables better governance and data model management by allowing you to track changes and updates to models over time. dbt_project.yml versions refer to the compatibility of the dbt project with a specific version of dbt. Version numbers within .yml property files informs how dbt parse those YAML files. The latter two are completely optional starting from version 1.5.

Author: bloody hell, i really confused model versions with yml versions 😬😅 - i love your proposal @dbeatty10 , thanks for the knowledge bombs and I'll go ahead and add your suggs and tag you for a last review if that's ok!

> :::info Model versioning is different to a version tag

Author: ```suggestion
:::info Model versions, dbt_project.yml versions, and .yml versions
```

Take note that [model versions](/docs/collaborate/govern/model-versions) are different from [dbt_project.yml versions](/reference/project-configs/version#dbt_projectyml-versions) and [.yml property file versions](/reference/project-configs/version#yml-property-file-versions).

> Model versions is a _feature_ that enables better governance and data model management by allowing you to track changes and updates to models over time. dbt_project.yml versions refer to the compatibility of the dbt project with a specific version of dbt. Version numbers within .yml property files inform how dbt parses those YAML files. The latter two are completely optional starting from version 1.5.

Author: ```suggestion
Model versions is a _feature_ that enables better governance and data model management by allowing you to track changes and updates to models over time. dbt_project.yml versions refer to the compatibility of the dbt project with a specific version of dbt. Version numbers within .yml property files inform how dbt parses those YAML files. The latter two are completely optional starting from dbt v1.5.
```

:::info Model versioning is different to a version tag

Take note that [model versions](/docs/collaborate/govern/model-versions) are different from [dbt_project.yml versions](/reference/project-configs/version#dbt_projectyml-versions) and [.yml property file versions](/reference/project-configs/version#yml-property-file-versions).

Model versions is a _feature_ that enables better governance and data model management by allowing you to track changes and updates to models over time. dbt_project.yml versions refer to the compatibility of the dbt project with a specific version of dbt. Version numbers within .yml property files inform how dbt parses those YAML files. The latter two are completely optional starting from version 1.5.

> Model versions and `versions` are _features_ that enable better governance and data model management by allowing you to track changes and updates to models over time.  Model versions and `versions` are available in dbt v1.5 or higher.

Author: ```suggestion
```
